core/backend/src/BlobContainerService.ts

```diff
@@ -45,26 +45,21 @@ export namespace BlobContainer {
     iTwinId: Id64String;
     /** optionally, an iModelId within the iTwin. If present, container is deleted when the iModel is deleted. */
     iModelId?: Id64String;
-    /** optionally, an owner of the container. Owners always have administrator rights for the container. */
-    owner?: string;
   }
 
   /**
    * Metadata about the use of a container so that:
    *  - administrators can understand why a container exists for assigning RBAC permissions appropriately
    *  - usage reports can aggregate types of containers
    *  - applications can identify their containers
-   *  - applications can store important (to them) properties "on" their containers
    */
   export interface Metadata {
-    /** Human readable explanation of the information held in the container. This will be displayed in the administrator RBAC panel, and on usage reports. */
-    description: string;
-    /** a machine-readable string that describes the "format" of the data in this container (e.g. "CloudSqlite") */
-    format: string;
-    /** an identifier of the application that uses this this container */
-    application: string;
-    /** Additional properties stored on the container */
-    [propertyName: string]: string;
```

Comment:
Why remove this? That means it won’t allow “unrecognized” members. I thought the idea was that people should be able to store their own Metadata, no?

Reply:
I'm not against the idea of additional metadata, but I'd like to understand the use cases a bit better before we commit to supporting it in the API...

I mean, you can always store other metadata _within_ your containers.  And if we discover that consumers prefer using the service to track certain things, it should be very quick and easy for us to add additional metadata fields.

Reply:
The idea of metadata is that you can read it without an access token. Here’s an example of something I think would be useful. The concepts of “purging” and “vacuuming” are things that average clients won’t know when to do. I imagine an agent that scans through containers with the appropriate metadata and performs one of those actions as a hygiene action periodically. Probably other more specific tasks that can be performed on specially tagged containers.

Reply:
Per our discussion yesterday, the service will continue to dictate the schema metadata for the time being - as new use cases for these fields come up @MichaelSwigerAtBentley will add them to the service as well as the type here.